Installation/changes.html

```diff
@@ -260,6 +260,13 @@ <h2 id="release4.10">Release 4.10 </h2>
 
 
 <!-- Surface Reconstruction -->
+  <h3>Poisson Surface Reconstruction</h3>
+  <ul>
+    <li>A new global
+    function <code>CGAL::poisson_surface_reconstruction_delaunay()</code>
+    is provided in addition to the current class-based API in order to
+    make it easier to use.</li>
+  </ul>
```

Comment:
That branch seems rather old, and this section of the changelog is in the changes of CGAL-4.10!